prow/scripts/lib/testing-helpers.sh

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env bash
```

Comment:
This file has different permissions than others in the same dir (its not executable).

Reply:
fixed